dashboards/nrm4sap/sap-dashboard.json

```diff
@@ -0,0 +1,2748 @@
+{
+  "name": "SAP Dashboards",
+  "description": "Monitor SAP systems and applications performances",
+  "pages": [
+    {
+      "name": "Index",
+      "description": null,
+      "widgets": [
+        {
+          "title": "SAP Systems",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 5,
+            "height": 6
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT uniqueCount(INSTANCE) as 'Num of Instaces' WHERE metricName = 'NR.SAP.SYSTEM.CPU_UTILIZATION' SINCE 30 minutes ago FACET SYS_ID"
```

Comment:
```suggestion
                "query": "FROM Metric SELECT uniqueCount(INSTANCE) as 'Num of Instances' WHERE metricName = 'NR.SAP.SYSTEM.CPU_UTILIZATION' SINCE 30 minutes ago FACET SYS_ID"
```